dockers/base-cuda/Dockerfile

```diff
@@ -136,6 +136,13 @@ RUN \
     if [[ "$CUDA_VERSION_MM" = "$CUDA_VERSION_BAGUA" ]]; then python -c "import bagua_core; bagua_core.install_deps()"; fi && \
     python -c "import bagua; print(bagua.__version__)"
 
+RUN \
+    # install ColossalAI
+    CUDA_VERSION_MM_COLOSSALAI=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda)))") && \
+    CUDA_VERSION_COLOSSALAI=$(python -c "print([ver for ver in [11.3, 11.1] if $CUDA_VERSION_MM_COLOSSALAI >= ver][0])") && \
+    pip install "colossalai==0.1.10+torch1.12cu${CUDA_VERSION_COLOSSALAI}" --find-links https://release.colossalai.org && \
```

Comment:
Please, also parametrize torch version. I know we are using currently 1.12, but it would be ideal to set and forget, so when the update to 1.13 happens, we don't have to change it here

Reply:
done